gpu/install_gpu_driver.sh

```diff
@@ -1752,13 +1755,14 @@ function prepare_gpu_env(){
       "1BB3" ) gpu_type="nvidia-tesla-p4"        ;;
       "1DB1" ) gpu_type="nvidia-tesla-v100"      ;;
       "1EB8" ) gpu_type="nvidia-tesla-t4"        ;;
-      "20B2" | \
```

Comment:
although this syntax may work, let's just duplicate code to keep things simpler.